tensorflow/core/ops/nn_ops.cc

```diff
@@ -2884,6 +2884,7 @@ REGISTER_OP("QuantizedConv2DWithBiasSignedSumAndReluAndRequantize")
       TF_RETURN_IF_ERROR(c->WithRank(c->input(4), 0, &unused));
       TF_RETURN_IF_ERROR(c->WithRankAtMost(c->input(5), 1, &channel));
       TF_RETURN_IF_ERROR(c->WithRankAtMost(c->input(6), 1, &channel));
+      // TODO(bhavanis): Need to use c->WithRankAtMost(...) for 7th input?
```

Comment:
Should the shapes of `min_output` and `max_output` here be set to `channel`?

Reply:
They are scalar since activations are not requantized per channel. I have added this explanation as a comment in the code.

Reply:
Sounds great. Thank you!